kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/DescrNameViewModel.java

```diff
@@ -79,50 +78,32 @@ public DescrNameViewModel() {
         ;
     }
 
-    public Set<ConceptEntity> findAllLanguages(ViewProperties viewProperties) {
-        IntIdSet languageDescendents = viewProperties.calculator().descendentsOf(TinkarTerm.LANGUAGE.nid());
-        Set<ConceptEntity> allLangs = languageDescendents.intStream()
-                .mapToObj(langNid -> (ConceptEntity) Entity.getFast(langNid))
-                .collect(Collectors.toSet());
-        return allLangs;
+    public Set<ConceptEntity> findAllLanguages() {
```

Comment:
Can we just remove this function?
The caller can call the `fetchDescendentsOfConcept()` directly.